.github/workflows/windows-amd64.yml

```diff
@@ -0,0 +1,655 @@
+          ls -hal
+          popd
+      - uses: actions/download-artifact@v2
+        with:
+          name: "swig_Windows"
+          path: ${{ github.workspace }}/native_client/ds-swig/
+      - name: Link ds-swig into swig
+        run: |
+          set -ex
+          ls -hal native_client/ds-swig/bin
+          ln -s ds-swig.exe native_client/ds-swig/bin/swig.exe
+          chmod +x native_client/ds-swig/bin/ds-swig.exe native_client/ds-swig/bin/swig.exe
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+      - uses: ./.github/actions/node-build
+        with:
+          nodejs_versions: "10.0.0 11.0.0 12.7.0 13.0.0 14.0.0 15.0.0"
+          electronjs_versions: "5.0.13 6.0.12 6.1.7 7.0.1 7.1.8 8.0.1 9.0.1 9.1.0 9.2.0 10.0.0 10.1.0 11.0.0 12.0.0"
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "nodewrapper-${{ matrix.build-flavor }}.tar.gz"
+          path: ${{ github.workspace }}/native_client/javascript/wrapper.tar.gz
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "deepspeech-${{ matrix.build-flavor }}.tgz"
+          path: ${{ github.workspace }}/native_client/javascript/deepspeech-*.tgz
+  test-cpp-Windows:
+    name: "Test C++ binary on Windows"
+    runs-on: windows-2019
+    needs: [build-lib_Windows, train-test-model]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+    env:
+      TASKCLUSTER_TMP_DIR: tmp/
+      DEEPSPEECH_TEST_MODEL: tmp/output_graph.pb
+      EXPECTED_TENSORFLOW_VERSION: "TensorFlow: v2.3.0-6-g23ad988"
+    steps:
+      - name: Switch git-bash shell to MSYS2 shell by adding MSYS2 path to PATH front
+        run: echo "D:\a\_temp\msys\msys64\usr\bin" >> $GITHUB_PATH
+      - uses: mozilla/setup-msys2@v2
+        with:
+          msystem: MSYS
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Download native_client.tar.xz
+        uses: actions/download-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.tar.xz"
+          path: ${{ env.TASKCLUSTER_TMP_DIR }}
+      - name: Extract native_client.tar.xz
+        run: |
+          mkdir -p ${{ env.TASKCLUSTER_TMP_DIR }}/ds
+          pushd ${{ env.TASKCLUSTER_TMP_DIR }}/ds
+          "C:/Program Files/7-Zip/7z.exe" x ../native_client.tar.xz -so | "C:/Program Files/7-Zip/7z.exe" x -aoa -si -ttar -o`pwd`
+          popd
+      - name: Download trained test model
+        uses: actions/download-artifact@v2
+        with:
+          name: "test-model.${{ matrix.build-flavor }}-16k.zip"
+          path: ${{ env.TASKCLUSTER_TMP_DIR }}
+      - run: |
+          ls -hal ${{ env.TASKCLUSTER_TMP_DIR }}/
+      - uses: ./.github/actions/run-tests
+        with:
+          runtime: "cppwin"
+          build-flavor: ${{ matrix.build-flavor }}
+          bitrate: "16k"
+          model-kind: ""
```

Comment:
I think we should at least have C++ tests passing with prod model, not just tests model